travis-install.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-rime_version=1.7.2
+rime_version=1.7.1
```

Comment:
CI will fails without this change because `librime@1.7.2` is not published on GitHub. @lotem Can you trigger a release? I will rebase once it is published.

Reply:
the osx build for librime 1.7.2 on Travis CI didn't finish. maybe it was a CI platform issue. retrying.  I'll publish the release when the build is done.

Reply:
Done.